components/cli/vendor/github.com/docker/docker/api/types/swarm/secret.go

```diff
@@ -12,7 +12,8 @@ type Secret struct {
 // SecretSpec represents a secret specification from a secret in swarm
 type SecretSpec struct {
 	Annotations
-	Data []byte `json:",omitempty"`
+	Data   []byte  `json:",omitempty"`
```

Comment:
Brought in by this PR:
* https://github.com/moby/moby/pull/34123 pluggable secret backend